var/spack/repos/builtin/packages/namd/package.py

```diff
@@ -103,9 +108,15 @@ class Namd(MakefilePackage, CudaPackage, ROCmPackage):
     depends_on("tcl", when="interface=python")
     depends_on("python", when="interface=python")
 
-    conflicts("+avxtiles", when="@:2.14,3:", msg="AVXTiles algorithm requires NAMD 2.15")
     conflicts("+rocm", when="+cuda", msg="NAMD supports only one GPU backend at a time")
     conflicts("+single_node_gpu", when="~cuda~rocm")
+    conflicts("+avxtiles", when="@:2.14", msg="AVXTiles algorithm requires NAMD 2.15+")
+    conflicts("+memopt", when="@:2.8", msg="memopt mode requires NAMD 2.8+")
+    conflicts(
+        "+memopt",
+        when="+single_node_gpu",
+        msg="memopt mode is not compatible with GPU-resident builds",
```

Comment:
Does Python ignore extra commas at the end of argument lists?

Reply:
black inserts those trailing commas and we fail style checks without them

Reply:
I see. Apparently this is actually recommended practice.